src/pushsource/_impl/list_cmd.py

```diff
@@ -59,6 +63,7 @@ def add_enum_representers(cls):
 
 ItemDumper.add_enum_representers()
 ItemDumper.add_representer(frozendict, ItemDumper.represent_dict)
+ItemDumper.add_representer(type(lambda: None), ItemDumper.represent_callable)
```

Comment:
I would prefer to just filter these out rather than dumping them. (i.e. do not show `opener` field here, same as we don't show it in `repr`.)

Firstly because there is no way `opener` can survive a round-trip anyway, since we only serialize the callable's name and not the code.  I'm pretty sure prior to this change, you can take the output of "pushsource-ls" and paste it into a python file and it'll actually work to construct push items, but that will be broken by this.

On top of that, this leaks some internal implementation details. `open_src_local` isn't public API, but its name is going to become visible here if we don't filter it out. If people actually use the serialized `opener` field for something, that can potentially block us from renaming or refactoring it later.